moths/scripts/valid_data_paths.py

```diff
@@ -0,0 +1,57 @@
+from pathlib import Path
```

Comment:
How and when is this script used?

Reply:
You remember that starting a train run took 99 hours because it has to loop over all files and check if they are valid for every run? I thought it would be nice if we do that only once. 

Run once, created file, give file in config. Then override the `is_valid_file` function to check if it is in this list.